Gateway/Request/MotoMerchantAccountDataBuilder.php

```diff
@@ -12,14 +12,13 @@
 namespace Adyen\Payment\Gateway\Request;
 
 use Adyen\AdyenException;
+use Magento\Payment\Gateway\Data\PaymentDataObject;
 use Magento\Payment\Gateway\Request\BuilderInterface;
 
 class MotoMerchantAccountDataBuilder implements BuilderInterface
 {
-    /**
-     * @var \Adyen\Payment\Helper\Requests
-     */
-    private $adyenRequestsHelper;
+
+    private \Adyen\Payment\Helper\Requests $adyenRequestsHelper;
```

Comment:
Could you please you import alias instead of full path for this and all similar examples?